homeassistant/components/template/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "template",
   "name": "Template",
   "documentation": "https://www.home-assistant.io/integrations/template",
-  "codeowners": ["@PhracturedBlue", "@tetienne"],
+  "codeowners": ["@PhracturedBlue", "@tetienne", "@home-assistant/core"],
```

Comment:
You can even remove me from the list. I just clean and add color to the light template. But do not really own the component.

Reply:
@tetienne can you make a separate PR to remove yourself if it is needed? If not if you would like me to make one for you let me know. There is no problem that you stay here as a code owner, unless it disturbs you getting notifications for it :-)